checker/src/constant_domain.rs

```diff
@@ -274,7 +274,11 @@ impl ConstantDomain {
                 if *val2 == 0 {
                     ConstantDomain::Bottom
                 } else {
-                    ConstantDomain::I128(*val1 / *val2)
+                    let result = val1.checked_div(*val2);
```

Comment:
` val1.checked_div(*val2).map(ConstantDomain::I128).map_err(ConstantDomain::Bottom)`

does that work?

Reply:
Nope, but I guess you really meant:
`val1.checked_div(*val2).map(ConstantDomain::I128).unwrap_or(ConstantDomain::Bottom)`